promoted_python_delivery_client/model/cohort_arm.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+
+class CohortArm(Enum):
+    UNKNOWN = 0
+    CONTROL = 1
```

Comment:
Side - I think promoted-ts-client has a bug with the checks.  I think it checks just TREATMENT.

Reply:
Yeah, I always had a nagging feeling if and when we use the other enum values, we'll have to sweep some codebases to make sure they're handled correctly.